### PR TITLE
Update chart guidelines to include interactive chart legends

### LINKED ISSUE #1533: Update Charts design guidelines to include information on interactive legends
The Charts design guideline documentation here should be updated to include information on behavior and usage of interactive legends. https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/charts

### PR DESCRIPTION
**Pull request by nlcwong**
Closes #1533 .

### CODE REVIEW

File: packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/charts/charts.md [hunk 1]
```diff
@@ -200,3 +200,19 @@ The brush and zoom properties allow for magnification of an area, bar, line or s
 
 1. **Window:** Click and drag to create a window around the section of the chart that you want to zoom in on
 2. **Magnified chart:** The portion selected will appear above the full chart view.
+
+### Interactive chart legends
+For charts that have more than one data set graphed, interactive legends can be used to highlight or isolate certain data sets.  This would work well for [line](#line-chart), [area](#area-chart), or [stacked](#stacked-bar-chart) type of charts where overlap of data sets could occur.
```
Comment (gdoyle1) [inline packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/charts/charts.md]: @nlcwong Add an "s" to the end of #stacked-bar-chart ! So it'll be [stacked](#stacked-bar-charts)

Comment (nlcwong) [reply packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/charts/charts.md]: I just updated the file to have the #stacked-bar-charts corrected.

**Comment by mcarrano** [comment]
@nlcwong Looks good to me. @gdoyle1 once you approve this, I can go ahead and merge.

**Comment by mcarrano** [comment]
Thanks for your contribution on this @nlcwong . I will merge!

**Pull request merged**